### DISCUSSION TITLE
getPeakSet output question

### DISCUSSION THREAD

**Discussion by yangxueqianR** [GreenleafLab/ArchR#1484] [Questions / Documentation] [answered]
Hello ,
I have some question when I get PeakSet after call peak. I am interesting in differences of peaks in different developmental stages. So I do **addReproduciblePeakSet** group by different time points. When I **getPeakSet**, I get the following result.
```R
ranges(getPeakSet(proj))
IRanges object with 327671 ranges and 0 metadata columns:
              start       end     width
          <integer> <integer> <integer>
    10hpf      3346      3846       501
    12hpf      3922      4422       501
    18hpf      4855      5355       501
    18hpf      5651      6151       501
  5.25hpf      6276      6776       501
      ...       ...       ...       ...
    18hpf  37492285  37492785       501
    12hpf  37492832  37493332       501
  5.25hpf  37497218  37497718       501
    10hpf  37498504  37499004       501
    12hpf  37499315  37499815       501
```
So I wonder what the rowname means, does it mean that this peak is prevalent at this time of development?
Thanks for you help!

**Comment by rcorces** [comment, marked as answer]
https://www.archrproject.com/bookdown/calling-peaks-w-macs2.html#:~:text=To%20retrieve%20this,that%20peak%20call.

**Reply by yangxueqianR** [reply]
Thank you for your reply，then if annotation is highest normalized significance peak origination for that peak call for the group，what is this difference from the following ***getMarkerFeature***.

**Reply by rcorces** [reply]
Just because something is highest, doesnt mean that another cell group isnt nearly just as highly significant. getMarkerFeatures looks for differential signal.

**Reply by yangxueqianR** [reply]
Ok, I see. Thank you for your patience.
